src/RadixDlt.NetworkGateway.GatewayApi/Validators/StateEntityFungibleResourceVaultsPageRequestValidator.cs

```diff
@@ -87,6 +87,11 @@ public StateEntityFungibleResourceVaultsPageRequestValidator(IOptionsSnapshot<En
         RuleFor(x => x.Cursor)
             .Base64();
 
+        RuleFor(x => x.AtLedgerState)
```

Comment:
This could be easily improved using following setup:
```
When(() => string.Isnullorempty(x.Cursor))
  .Then(() => {
    RuleFor(x => x.Cursor).NotEmpty().Base64();
    RuleFor(x => x.LedgerState).NotNull().HasStateVersion();
  })
  .Else(() => {
    RuleFor(x => x.Cursor).empty();
    RuleFor(x => x.LedgerState).SetValidator(....);
 })
```